packages/core/src/lib/list-box/__tests__/list-box.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import { ListBox } from '$lib';
+import { default as ListBoxSlot } from '../list-box-slot.svelte';
+
+const defaultProps = {
+  disabled: false,
+  left: 'leftText1,leftText2,leftText3',
+  right: 'rightText1,rightText2,rightText3',
+  leftlabel: 'Left',
+  rightlabel: 'Right',
+  height: '200px',
+  suffix: false,
+};
+
+describe('ListBox', () => {
+  it('Renders left options and can move them to the right', async () => {
+    const user = userEvent.setup();
+    render(ListBoxSlot, { ...defaultProps });
+
+    await user.click(screen.getByText('leftText1'));
```

Comment:
When we're clicking elements in tests, we should always use `getByRole('button', ...)` (or whatever similar role). `getByText` should be your third or fourth choice when you're trying to pick an element query in a test